qrpage.md

```diff
@@ -7,3 +7,5 @@ photos:
   license: Unsplash License
```

Comment:
could you maybe use `qrpage` as layout here instead of the include?

Reply:
Oh yeah...sure.